install.md

```diff
@@ -250,6 +252,8 @@ To delete the remaining resources run:
 kubectl delete pods,serviceaccount,clusterrolebinding --selector kritis.grafeas.io/install --namespace <your namespace>
 ```
 
+If you used the default namespace during the installation, you don't need to provide the `--namespace <your namespace>`.
```

Comment:
minor nit - feel free to disagree/ignore: I feel like this could be improved so that:

- We don't ask users to rewrite the command line they've pasted based on something they may not be sure about.
- We clarify before they are likely to paste the command in.

Two ideas:

"To delete the remaining resources, run the following, where `<namespace>` is `default`, or the name you specifically selected at installation time:
..."

Or perhaps mentioning afterwards in such a way that simply has them plug the appropriate value into --namespace rather than omitting it:

If you did not specifically select a namespace during installation, the default value is `default`.

Thoughts?

Reply:
That's a great point. I went with your second suggestion, as I think it sheds some light into the internals that weren't mentioned before, while keeping it as a side note.